### ISSUE TITLE
Tag v1.0.420 references develop

### ISSUE THREAD

**Issue by ARH-Digital** [octobercms/october#3082]
##### Expected behavior
Ability to install october via CLI

##### Actual behavior
Install fails due to ``Your requirements could not be resolved to an installable set of packages.`` As the composer.json is still referencing dev-develop and @dev packages

##### Reproduce steps
Create a new project from composer as per documentation with the ``composer create-project october/october myoctober``

##### October build
October build 420

Bad references - ``https://github.com/octobercms/october/blob/v1.0.420/composer.json``
Correct references - ``https://github.com/octobercms/october/blob/master/composer.json``

**Comment by morph85** [comment]
I'm having the same issue performing "composer update" too.
I created project the same way using composer and developed over release 1.0.420.
However with composer update command I get this:

```
$ composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for october/system dev-develop -> satisfiable by october/system[dev-develop].
    - october/system dev-develop requires october/rain dev-l55upgrade -> no matching package found.
  Problem 2
    - Installation request for october/backend dev-develop -> satisfiable by october/backend[dev-develop].
    - october/backend dev-develop requires october/rain dev-l55upgrade -> no matching package found.
  Problem 3
    - Installation request for october/cms dev-develop -> satisfiable by october/cms[dev-develop].
    - october/cms dev-develop requires october/rain dev-l55upgrade -> no matching package found.

Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.

Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

By using the composer.json with correct reference I'm able to update again.

**Comment by morph85** [comment]
However the composer.json above still reference build 1.0.419.

**Comment by daftspunk** [comment]
Refs #3081

**Comment by LukeTowers** [comment]
Closed since #3081 is resolved.
